kythe/javatests/com/google/devtools/kythe/analyzers/java/testdata/pkg/DefaultCorpus.java

```diff
@@ -12,6 +12,8 @@
   //- ListStringType.node/kind "tapp"
   private List<String> myvar;
 
+  private List<? extends Object> objList;
```

Comment:
This appears unused?

Reply:
I added some verifier assertions here, which show the intended change. The TAppNode now has corpus "kythe" instead of empty string.